docs/index.rst

```diff
@@ -40,6 +40,19 @@ encoding
   Encoding to be used to read an OpenAPI spec. If not passed, Sphinx's
   source encoding will be used.
 
+paths
+  A comma separated list of paths to filter the included openapi spec by. For example:
```

Comment:
Whoops. Yeah I'll fix that up. My editor is set to 100 by default, so that's my bad.